code/drasil-printers/lib/Language/Drasil/Printing/Import/ModelExpr.hs

```diff
@@ -118,8 +118,8 @@ modelExpr (Lit l)                    sm = literal l sm
 modelExpr (AssocB And l)             sm = assocExpr P.And (precB And) l sm
 modelExpr (AssocB Or l)              sm = assocExpr P.Or (precB Or) l sm
 modelExpr (AssocB Equivalence l)     sm = assocExpr P.Eq (precB Equivalence) l sm
-modelExpr (AssocA AddI l)            sm = assocExpr P.Add (precA AddI) l sm
-modelExpr (AssocA AddRe l)           sm = assocExpr P.Add (precA AddRe) l sm
+modelExpr (AssocA AddI l)            sm = P.Row $ addExpr l AddI sm
+modelExpr (AssocA AddRe l)           sm = P.Row $ addExpr l AddRe sm
```

Comment:
@JacquesCarette This is the change for `AssocA AddRe/AddI` in the imported phrase. Instead of using `assocExpr` which adds the operator between items in the list, it will use a new create function `addExpr` to pattern match the negation sign and only add `+` once the second part is not negation. 

I think my question is which import phase we would like to have this change. There are three files in the import, CodeExpr.hs, Expr.hs, and ModelExpr.hs. They almost look identical with each other. They all have `AssocA AddRe/AddI`

The current change made in import/ModelExpr.hs

Reply:
The change should be done to all three.